### PR TITLE
new resource `azurerm_spring_cloud_build_deployment`

### PR DESCRIPTION
**Pull request by ms-henglu**
image

### CODE REVIEW

File: vendor/github.com/Azure/azure-sdk-for-go/services/network/mgmt/2021-08-01/network/_meta.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "commit": "fc9d4d2798f755bea848ac1c29b2730d31002cb8",
+  "commit": "50ed15bd61ac79f2368d769df0c207a00b9e099f",
```
Comment (katbyte) [inline vendor/github.com/Azure/azure-sdk-for-go/services/network/mgmt/2021-08-01/network/_meta.json:2]: why is this changing?

File: website/docs/r/spring_cloud_build_deployment.html.markdown [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+* `quota` - (Optional) A `quota` block as defined below.
+
+---
+
+A `quota` block supports the following:
+
+* `cpu` - (Optional) Specifies the required cpu of the Spring Cloud Deployment. Possible Values are `500m`, `1`, `2`, `3` and `4`. Defaults to `1` if not specified.
+
+-> **Note:** `cpu` supports `500m` and `1` for Basic tier, `500m`, `1`, `2`, `3` and `4` for Standard tier.
```
Comment (katbyte) [inline website/docs/r/spring_cloud_build_deployment.html.markdown:79]: can we add a check to the create function for this?

File: website/docs/r/spring_cloud_build_deployment.html.markdown [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+* `memory` - (Optional) Specifies the required memory size of the Spring Cloud Deployment. Possible Values are `512Mi`, `1Gi`, `2Gi`, `3Gi`, `4Gi`, `5Gi`, `6Gi`, `7Gi`, and `8Gi`. Defaults to `1Gi` if not specified.
+
+-> **Note:** `memory` supports `512Mi`, `1Gi` and `2Gi` for Basic tier, `512Mi`, `1Gi`, `2Gi`, `3Gi`, `4Gi`, `5Gi`, `6Gi`, `7Gi`, and `8Gi` for Standard tier.
```
Comment (katbyte) [inline website/docs/r/spring_cloud_build_deployment.html.markdown:83]: and here

**Review by katbyte — APPROVED** [review]
LGTM ⭐

File: vendor/github.com/Azure/azure-sdk-for-go/services/network/mgmt/2021-08-01/network/_meta.json (hunk 1 above)
Comment (katbyte) [reply vendor/github.com/Azure/azure-sdk-for-go/services/network/mgmt/2021-08-01/network/_meta.json:2]: why is this changing?

**Review by katbyte — CHANGES_REQUESTED** [review]

**Comment by ms-henglu** [comment]
Hi @katbyte , I've removed the unnecessary changes, I think they're introduced at a time there's vendor diff.

**Review by katbyte — APPROVED** [review]
LGTM 👁️‍🗨️

**Pull request merged**